_candidates_en/TakeshiItoh_1.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Takeshi Itoh @iTakeshi
+title: "Metaprogramming in Scala: the Past and the Present"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: TaKeZo_I
+github: iTakeshi
+icon: https://avatars2.githubusercontent.com/u/1460175
+organization: Robotic Biology Institute Inc., Nara Institute of Science And Technology
+tags:
+  - Tools
+suggestions:
+  - People who want to try metaprogramming using Scala
+  - People who want to understand internal of libraries using metaprogramming
```

Comment:
suggested change "+  - People who want to understand the internal workings of libraries using metaprogramming"